server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTree.java

```diff
@@ -1718,105 +1758,96 @@ public int getMountedNodeIndexOnMeasurementPath(PartialPath measurementPath)
     return fullPathNodes.length - 1;
   }
 
-  public List<String> getPathsSetOnTemplate(String templateName) throws MetadataException {
+  public List<String> getPathsSetOnTemplate(Template template) throws MetadataException {
+    String templateName = template == null ? "" : template.getName();
```

Comment:
Should this template name "" replaced by * ? I see * means all template.

Reply:
fixed now